addons/binding/org.openhab.binding.velux/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: velux Binding
+Bundle-SymbolicName: org.openhab.binding.velux;singleton:=true
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
```

Comment:
Please rebase your commit on the current master and use version 2.2* where it applies.

Reply:
Done.